external/cmn/bundle_client.go

```diff
@@ -280,7 +280,7 @@ func (c *BundleClient) UploadObject(fileName, bucketName, bundleName, contentTyp
 	return nil
```

Comment:
When will be the method "func (c *BundleClient) UploadObject"  used?  Seemingly it is not being used now.

Reply:
Currently, I use the function UploadAndFinalizeBundle(bundleName, bucketName, bundleDir, bundlePath string) to upload to Greenfield via the bundle service. This function is now obsolete and has been removed.